.github/workflows/release.yml

```
@@ -9,11 +9,11 @@ jobs:
   build-and-upload:
     runs-on: macos-15
     env:
-      DEVELOPER_DIR: /Applications/Xcode_16.app/Contents/Developer
+      DEVELOPER_DIR: /Applications/Xcode_16.4.app/Contents/Developer
 
     steps:
       - name: Select Xcode 16
-        run: sudo xcode-select -s /Applications/Xcode_16.app
+        run: sudo xcode-select -s /Applications/Xcode_16.4.app
```

Please use 


```suggestion
        run: sudo xcode-select -s /Applications/Xcode.app
```

It will be 16.4 as well but auto-upgrade in the future.